e2e/rbd.go

```diff
@@ -115,6 +115,16 @@ var _ = Describe("RBD", func() {
 			By("create a PVC and Bind it to an app with normal user", func() {
 				validateNormalUserPVCAccess(pvcPath, f)
 			})
+			// Skipping ext4 FS testing
+
+			By("create a PVC and Bind it to an app with ext4 as the FS ", func() {
+				deleteResource(rbdExamplePath + "storageclass.yaml")
```

Comment:
(future) We could have created a new storage class (with `ext4` as the fsType) instead of deleting and recreating the older storage class. Helps with possible(!?) future tests where we may want to test with different storage class parameters. (It is more change to other testing APIs as well, so leave it out for now)

Reply:
actually it was the case in first PR, however removed as per review comment. But yeah, we can bring it on future PRs. Thanks!